### PR TITLE
feat: Add Background component and stories #P23-118

### PR DESCRIPTION
**Pull request by Noshi96**
Ticket: https://tracker.intive.com/jira/browse/PATRO23-118
Converter: https://base64.guru/converter/encode/image/svg
SVG-Minifier: https://devina.io/svg-minifier

### CODE REVIEW

File: packages/ui/Background/index.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
...xNjAuNDUxIDE0Ni42NjEgMTU2LjgxOCAxNTEuMjQ1IDE1OS4zMjlMMTU2LjAzOSAxNjEuOTU2QzE2MC4zNTMgMTY0LjMxOSAxNjAuNTk4IDE3MC40MjQgMTU2LjQ4NyAxNzMuMTI1TDE1Mi4yMDIgMTc1Ljk0MkMxNDguMDkxIDE3OC42NDMgMTQyLjU4NCAxNzUuOTk1IDE0Mi4xMjcgMTcxLjA5OEwxNDEuNjE5IDE2NS42NTVaIiBmaWxsPSIjNjRCQTk1Ii8+CjxwYXRoIGZpbGwtcnVsZT0iZXZlbm9kZCIgY2xpcC1ydWxlPSJldmVub2RkIiBkPSJNMzE3Ljk1MSA5Ni42NDUzTDMzMS4zMTEgMTI5LjU4NkwyODMuMDg0IDEwMy41NjZMMzE3Ljk1MSA5Ni42NDUzWk0zMjQuNTc1IDc0LjY0NTFDMzI3Ljk4NSA3My45Njg0IDMzMS4zOTQgNzUuODA4MSAzMzIuNzAxIDc5LjAyOTNMMzYzLjczOCAxNTUuNTU3QzM2Ni4yOTEgMTYxLjg1MSAzNTkuNTYgMTY3Ljg4MiAzNTMuNTgyIDE2NC42NTdMMjQxLjU0MyAxMDQuMjA3QzIzNS41NjYgMTAwLjk4MiAyMzYuOTExIDkyLjA0NDkgMjQzLjU3MyA5MC43MjI2TDMyNC41NzUgNzQuNjQ1MVoiIGZpbGw9IiM2NEJBOTUiLz4KPHBhdGggZmlsbC1ydWxlPSJldmVub2RkIiBjbGlwLXJ1bGU9ImV2ZW5vZGQiIGQ9Ik01NzUuNDIxIDU1My4zNTZMNjE5LjMxMyA1NzguMDM1TDYwMS4wOTQgNTEwLjAzN0w1NzUuNDIxIDU1My4zNTZaTTU1MS4xNjQgNTU0LjQ5MUM1NDkuMDkgNTU3Ljk4OSA1NTAuMzAxIDU2Mi41MDggNTUzLjg0NiA1NjQuNTAxTDYzNi4zODEgNjEwLjkwOEM2NDEuOTc0IDYxNC4wNTMgNjQ4LjU5MyA2MDguOTE0IDY0Ni45MzIgNjAyLjcxNkw2MTIuNjcxIDQ3NC44NTJDNjExLjAxMSA0NjguNjU1IDYwMi43MDkgNDY3LjUxNCA1OTkuNDM4IDQ3My4wMzNMNTUxLjE2NCA1NTQuNDkxWiIgZmlsbD0iIzQ1OTE3NSIvPgo8L2c+Cjwvc3ZnPgo=");
```
[the remark this answers is not in the document]
Comment (Noshi96) [reply packages/ui/Background/index.tsx]: I used svg minifier and managed to reduce the size by half. The question is, is this acceptable?

**Comment by aniasgit** [comment]
Looks good to me :)

**Review by aniasgit — APPROVED** [review]
Looks good to me :)

**Review by Alessiass — APPROVED** [review]
Looks fine for me, just food for thoughts if the children prop is necessary for background, still it's working as it should :).

**Pull request merged**